src/main.rs

```diff
@@ -1,26 +1,9 @@
-// Copyright 2016 Joe Wilm
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// 	http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 //! A tool for generating benchmark scripts for terminal emulators
 //!
 //! This program is intended to be run and its output piped into a file, and the
 //! file can simply be `cat`ed from the terminal emulator under test. This
 //! ensures that the terminal is being benchmarked and not this vtebench
 //! application.
-extern crate rand;
-extern crate structopt;
-
 #[macro_use]
```

Comment:
We should probably remove all the `#[macro_use]` annotations in favor of direct imports of the macros. To comply with the 2018 edition stuff.

Reply:
That's not a bad idea, I'll try to do so.